crates/proto/proto/zed.proto

```diff
@@ -1784,14 +1784,40 @@ message RepositoryEntry {
 
 message StatusEntry {
     string repo_path = 1;
-    GitStatus status = 2;
+    GitStatusCode simple_status = 2;
+    optional GitFileStatus status = 3;
```

Comment:
note: optional is superfluous; also add a note to drop `simple_status` at some point

Reply:
and add a note about the min version that will have the new `status` field here (so we can remove `simple_status` when the min version in collab is bumped)